src/components/explore/ProportionExplorerView.svelte

```diff
@@ -91,8 +88,8 @@ h2 {
       {/if}
     </div>
 
-    <div class=body-control-set>
-      <label class=body-control-set--label>Categories</label>
+    <div class="body-control-set">
```

Comment:
I'm seeing a lot of changes like this. Should we make this a lint rule so it is enforced? I have no preference but I typically don't put quotes in out of habit